README.md

```diff
@@ -1,71 +1,55 @@
 
 # Code Check #
 
-Do you have a problem with duplicated functions in your java project? Here is the solution on how to find all the duplicate functions, specify the location of your repository in the local-config.properties file (just make a copy of config.properties) and run the code! Then you will get a file in "results" with all the places where functions match.
+Do you have a problem with duplicated functions in your coding project? This might be the solution on finding them.
 
-### There are even some extra features such as ###
+### Features ###
 
-* query an LLM on complicated situations where 2 functions could be similar
-* lots of configuration
-* it even has testing for it!
+* Query an LLM (large language model) on complicated situations where 2 or more functions could be similar
+* Configuration - it's easy to change the configurations.
+* Tested on both UNIX and Windows.
 
 ## Testing ##
 
-### Manually ###
+To run the test files you can run this command with Gradle:
 
-To run the tests run this command:
-
-```bash
+```sh
```

Comment:
- [] Fungerar `sh` på Windows på samma sätt som i UNIX?

Reply:
Go for it